core/chains/evm/log/mocks/orm.go

```diff
@@ -15,20 +15,20 @@ type ORM struct {
 	mock.Mock
 }
 
-// CreateBroadcast provides a mock function with given fields: blockHash, blockNumber, logIndex, jobID, qopts
-func (_m *ORM) CreateBroadcast(blockHash common.Hash, blockNumber uint64, logIndex uint, jobID int32, qopts ...pg.QOpt) error {
+// CreateBroadcast provides a mock function with given fields: blockHash, blockNumber, txIndex, logIndex, jobID, qopts
+func (_m *ORM) CreateBroadcast(blockHash common.Hash, blockNumber uint64, txIndex uint, logIndex uint, jobID int32, qopts ...pg.QOpt) error {
```

Comment:
Is this `type ORM` unused, since we were still building and passing tests with mismatched method signatures?

Reply:
Probably new as this code was generated by `go generate`

Reply:
If it's unused, can we just remove it?

Reply:
I checked and it's being used, maybe the `qopts ...pg.QOpt` optional argument made it work before this PR? Also note this is a generated mock, so even if something is unused in the mock the original function could still be used.